### ISSUE TITLE
Color contrast on tags

### ISSUE THREAD

**Issue by phucduong86** [phucduong86/phucduong86.github.io#4]
Under each post title there are: date, tags and author.
This line doesn't meet minimum color contrast requirement.

**Comment by phucduong86** [comment]
Issue resolved.
